pallets/utxo/src/lib.rs

```diff
@@ -401,7 +536,7 @@ pub mod pallet {
             .ok_or("Sub underflow")
             .unwrap();
```

Comment:
Please avoid using unwrap. If this is an unrecoverable error and you want the program to crash, use expect() and explain in fair detail what happened and how our model is broken.